webapp/src/DOMJudgeBundle/Resources/views/jury/rejudging_form.html.twig

```diff
@@ -0,0 +1,23 @@
+{% extends "@DOMJudge/jury/base.html.twig" %}
+
+{% block title %}Add rejudging - {{ parent() }}{% endblock %}
+
+{% block content %}
+
+    <h1>Add rejudging</h1>
+
+    <div class="row">
+        <div class="col-lg-4">
+            {{ form_start(form) }}
```

Comment:
I think you output the whole form? If so, you can replace all this by
```twig
{{ form(form) }}
```

Reply:
That's easy :-D